miso-web/src/main/webapp/scripts/pool_ajax.js

```diff
@@ -52,8 +52,7 @@ var Pool = Pool || {
 
     // Description input field validation
     jQuery('#description').attr('class', 'form-control');
-    jQuery('#description').attr('data-parsley-required', 'true');
-    jQuery('#description').attr('data-parsley-maxlength', '100');
+    jQuery('#description').attr('data-parsley-maxlength', '255');
```

Comment:
I've just noticed that Pool volume is also set to `required`, would you mind changing that in this PR?